_includes/partial/site-footer.html

```diff
@@ -21,28 +21,29 @@
   <main class="footer-row">
     <div>
         <div class="footer-row-text-links">
-            <a class="" href="http://www.apache.org/">Apache&nbsp;Software&nbsp;Foundation</a>
+            <a class="" href="https://www.apache.org/">Apache&nbsp;Software&nbsp;Foundation</a>
             <a class="" href="https://apache.org/events/current-event">Events</a>
-            <a class="" href="http://www.apache.org/licenses/">License</a>
+            <a class="" href="https://www.apache.org/licenses/">License</a>
             <a class="" href="security.html">Security</a>
-            <a class="" href="http://www.apache.org/foundation/thanks.html">Thanks</a>
-            <a class="" href="http://www.apache.org/foundation/sponsorship.html">Sponsorship</a>
+            <a class="" href="https://www.apache.org/foundation/thanks.html">Thanks</a>
+            <a class="" href="https://www.apache.org/foundation/sponsorship.html">Sponsorship</a>
         </div>
     </div>
   </main>
   <main class="footer-row">
     <div>
       Apache OpenWhisk, OpenWhisk, Apache, the Apache feather logo, and
-      the Apache OpenWhisk project logo are either registered trademarks
-      or trademarks of The Apache Software Foundation
-      in the United States and other countries.
-      See guidance on use of Apache OpenWhisk trademarks.
+      the Apache OpenWhisk project logo are either registered
+      registered <a href="trademarks.html">trademarks</a> or
+      trademarks of The Apache Software Foundation in the United States and other countries.
+      See <a href="trademarks.html">guidance</a> on use of Apache OpenWhisk trademarks.
```

Comment:
Will this relative link work for pages which are not at the website root? It might - I don't know the details of how the site is generated.